frontend/bunfig.toml

```diff
@@ -0,0 +1,4 @@
+[install]
+save = true
+saveTextLockfile = true
+print = "yarn"
```

Comment:
We will always use text lockfile instead of binary, and we will use yarn lock to ensure compatibility with snyk security scan.